src/autocomplete/shared/tokenize.ts

```diff
@@ -7,7 +7,7 @@ export type Token = {
     ruleName: string;
     startIndex: number;
     line: number;
-    column: number;
+    columnIndex: number;
```

Comment:
I'd leave this as `column`
`columnIndex` doesn't make sense

Reply:
@viladimiru what do you think of this comment? I think `columnIndex` doesn't make much sense, because column already means index kind of